app_dart/lib/src/request_handlers/file_flaky_issue_and_pr.dart

```diff
@@ -143,13 +143,7 @@ class FileFlakyIssueAndPR extends ApiRequestHandler<Body> {
   }
 
   bool _getIgnoreFlakiness(String builderName, CiYaml ciYaml) {
-    final Target target;
-    try {
-      target = ciYaml.postsubmitTargets.singleWhere((Target target) => target.value.name == builderName);
-    } on StateError {
-      // Did not find a single target matching builderName
-      return false;
-    }
+    final Target target = ciYaml.postsubmitTargets.singleWhere((Target target) => target.value.name == builderName);
     return target.ignoreFlakiness();
```

Comment:
nit: target.getIgnoreFlakiness();

If this is const, you can make it a getter field

Reply:
done